tests/cli/commands/test_task_command.py

```diff
@@ -55,6 +55,7 @@ def reset(dag_id):
         runs.delete()
 
 
+# TODO: Check if tests needs side effects - locally there's missing DAG
```

Comment:
Can this be removed, looks like tests are passing

Reply:
That's the problem - they don't work locally when triggered without whole test suite

Reply:
I can test that and fix it if that is the case in a separate PR

Reply:
> I can test that and fix it if that is the case in a separate PR

Thanks for helping! The problems I observed:
```
root@01658a8a09d3:/opt/airflow# pytest -s tests/cli/commands/test_task_command.py
...
============================================================================================= short test summary info ==============================================================================================
SKIPPED [1] tests/conftest.py:313: The test is skipped because it has quarantined marker. And --include-quarantined flag is passed to pytest. <TestCaseFunction test_local_run>
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_list_tasks - airflow.exceptions.AirflowException: dag_id could not be found: example_bash_operator. Either the dag did not exist or it fai...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run - airflow.exceptions.AirflowException: dag_id could not be found: example_bash_operator. Either the dag did not exist or it failed to ...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_0__ignore_all_dependencies - AssertionError: "Option --raw does not work with some of the other options on this com...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_1__ignore_depends_on_past - AssertionError: "Option --raw does not work with some of the other options on this comm...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_2__ignore_dependencies - AssertionError: "Option --raw does not work with some of the other options on this command...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_3__force - AssertionError: "Option --raw does not work with some of the other options on this command." does not ma...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_mutually_exclusive - AssertionError: "Option --raw and --local are mutually exclusive." does not match "dag_id could not be found: exa...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test - airflow.exceptions.AirflowException: dag_id could not be found: example_bash_operator. Either the dag did not exist or it failed to...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test_with_env_vars - airflow.exceptions.AirflowException: dag_id could not be found: example_passing_params_via_test_command. Either the d...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test_with_params - airflow.exceptions.AirflowException: dag_id could not be found: example_passing_params_via_test_command. Either the dag...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_parentdag_downstream_clear - airflow.exceptions.AirflowException: dag_id could not be found: example_subdag_operator.section-1. Either the dag...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_subdag_clear - airflow.exceptions.AirflowException: dag_id could not be found: example_subdag_operator. Either the dag did not exist or it fai...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_task_state - airflow.exceptions.AirflowException: dag_id could not be found: example_bash_operator. Either the dag did not exist or it failed ...
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_task_states_for_dag_run - KeyError: 'example_python_operator'
FAILED tests/cli/commands/test_task_command.py::TestCliTasks::test_test - airflow.exceptions.AirflowException: dag_id could not be found: example_python_operator. Either the dag did not exist or it failed to p...
FAILED tests/cli/commands/test_task_command.py::TestLogsfromTaskRunCommand::test_logging_with_run_task - AssertionError: 2 != 1
=============================================================================== 16 failed, 6 passed, 1 skipped, 1 warning in 41.37s ================================================================================
```

This is what I get using brezee and current main branch.

Reply:
Weird:

```
❯ ./breeze --python 3.6 --backend postgres --db-reset
root@34129b165f1a:/opt/airflow# pytest tests/cli/commands/test_task_command.py::TestCliTasks
==================================================================================================================== test session starts =====================================================================================================================
platform linux -- Python 3.6.12, pytest-6.1.2, py-1.9.0, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow, configfile: pytest.ini
plugins: forked-1.3.0, timeouts-1.2.1, cov-2.10.1, celery-4.4.7, flaky-3.7.0, xdist-2.1.0, instafail-0.4.2, requests-mock-1.8.0, rerunfailures-9.1.1
setup timeout: 0.0s, execution timeout: 0.0s, teardown timeout: 0.0s
collected 17 items

tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_list_tasks PASSED                                                                                                                                                                      [  5%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run PASSED                                                                                                                                                                             [ 11%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_0__ignore_all_dependencies PASSED                                                                                                                               [ 17%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_1__ignore_depends_on_past PASSED                                                                                                                                [ 23%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_2__ignore_dependencies PASSED                                                                                                                                   [ 29%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_invalid_raw_option_3__force PASSED                                                                                                                                                 [ 35%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_run_mutually_exclusive PASSED                                                                                                                                                          [ 41%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test PASSED                                                                                                                                                                            [ 47%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test_with_env_vars PASSED                                                                                                                                                              [ 52%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_test_with_params PASSED                                                                                                                                                                [ 58%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_local_run SKIPPED                                                                                                                                                                          [ 64%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_parentdag_downstream_clear PASSED                                                                                                                                                          [ 70%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_run_naive_taskinstance PASSED                                                                                                                                                              [ 76%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_subdag_clear PASSED                                                                                                                                                                        [ 82%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_task_state PASSED                                                                                                                                                                          [ 88%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_task_states_for_dag_run PASSED                                                                                                                                                             [ 94%]
tests/cli/commands/test_task_command.py::TestCliTasks::test_test PASSED                                                                                                                                                                                [100%]

====================================================================================================================== warnings summary ======================================================================================================================
tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_list_tasks
  /usr/local/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)

tests/cli/commands/test_task_command.py::TestCliTasks::test_cli_list_tasks
  /usr/local/lib/python3.6/site-packages/sqlalchemy/sql/sqltypes.py:271: SAWarning: Unicode type received non-unicode bind param value "b'# -*- coding: utf-8 -*-\\n#'...". (this warning may be suppressed after 10 occurrences)
    (util.ellipses_string(value),),

tests/cli/commands/test_task_command.py::TestCliTasks::test_parentdag_downstream_clear
  /opt/airflow/airflow/models/dag.py:1195: DeprecationWarning: This method is deprecated and will be removed in a future version. Please use partial_subset
    include_downstream=True,

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================================================================================================================== short test summary info ===================================================================================================================
SKIPPED [1] tests/conftest.py:313: The test is skipped because it has quarantined marker. And --include-quarantined flag is passed to pytest. <TestCaseFunction test_local_run>
=================================================================================================== 16 passed, 1 skipped, 3 warnings in 165.93s (0:02:45) ====================================================================================================
```